src/frontend/Komponenter/Behandling/VedtakOgBeregning/Barnetilsyn/KontantstøtteValg.tsx

```diff
@@ -0,0 +1,183 @@
+    grid-template-areas: 'fraOgMedVelger tilOgMedVelger kontantstøtte slettKnapp';
+    grid-template-columns: ${(props) =>
+        props.lesevisning ? '8rem 10rem 7rem 7rem 7rem' : '12rem 12rem 6rem 4rem'};
+    grid-gap: ${(props) => (props.lesevisning ? '0.5rem' : '1rem')};
+    margin-bottom: 0.75rem;
+`;
+
+const KolonneHeaderWrapper = styled.div<{ lesevisning?: boolean }>`
+    display: grid;
+    grid-template-areas: 'fraOgMedVelger tilOgMedVelger kontantstøtte';
+    grid-template-columns: ${(props) =>
+        props.lesevisning ? '8rem 10rem 7rem' : '12rem 12rem 6rem'};
+    grid-gap: ${(props) => (props.lesevisning ? '0.5rem' : '1rem')};
+    margin-bottom: 0.5rem;
+`;
+
+const StyledInput = styled(InputMedTusenSkille)`
+    text-align: left;
+`;
+
+interface Props {
+    kontantstøttePerioder: ListState<IKontantstøttePeriode>;
+    valideringsfeil?: FormErrors<InnvilgeVedtakForm>['kontantstøtteperioder'];
+    settValideringsFeil: Dispatch<SetStateAction<FormErrors<InnvilgeVedtakForm>>>;
+}
+
+export const tomKontantstøtteRad: IKontantstøttePeriode = {
+    årMånedFra: '',
+    årMånedTil: '',
+    beløp: undefined,
+};
+
+const KontantstøtteValg: React.FC<Props> = ({
+    kontantstøttePerioder,
+    valideringsfeil,
+    settValideringsFeil,
+}) => {
+    const { behandlingErRedigerbar } = useBehandling();
+    const { settIkkePersistertKomponent } = useApp();
+    const [kontantstøtte, settKontantstøtte] = useState(false);
+
+    const oppdaterKontantstøttePeriode = (
+        index: number,
+        property: EKontantstøttePeriodeProperty,
+        value: string | string[] | number | undefined
+    ) => {
+        kontantstøttePerioder.update(
+            {
+                ...kontantstøttePerioder.value[index],
+                [property]: value,
+            },
+            index
+        );
+        settIkkePersistertKomponent(VEDTAK_OG_BEREGNING);
+    };
+
+    const periodeVariantTilKontantstøtteperiodeProperty = (
+        periodeVariant: PeriodeVariant
+    ): EKontantstøttePeriodeProperty => {
+        switch (periodeVariant) {
+            case PeriodeVariant.ÅR_MÅNED_FRA:
+                return EKontantstøttePeriodeProperty.årMånedFra;
+            case PeriodeVariant.ÅR_MÅNED_TIL:
+                return EKontantstøttePeriodeProperty.årMånedTil;
+        }
+    };
+
+    return (
+        <>
+            <RadioGruppe legend="Er det søkt om, utbetales det eller har det blitt utbetalt kontantstøtte til brukeren eller en brukeren bor med?">
+                <Radio
+                    name={'Kontantstøtte'}
+                    label={'Ja'}
+                    value={'Ja'}
+                    onChange={() => settKontantstøtte(!kontantstøtte)}
+                />
+                <Radio
+                    name={'Kontantstøtte'}
+                    label={'Nei'}
+                    value={'Nei'}
+                    checked={!kontantstøtte}
+                    onChange={() => settKontantstøtte(!kontantstøtte)}
```

Comment:
Bare sånn for sikkerhetsskyld - burde denne være `settKontantstøtte(false)` og den ovenfor `(true)` ? Dobbeltklikk kan kanskje bli litt moro her? :)

Reply:
Fikset i senere branch